addon/templates/components/layers-dialogs/settings/kml.hbs

```diff
@@ -0,0 +1,16 @@
+<div class="field">
+  <label>
+    {{t "components.layers-dialogs.settings.kml.url-textbox.caption"}}
+  </label>
+  {{flexberry-textbox
+    class=(concat "fluid" (if settings.kmlString " disabled" ""))
+    value=settings.kmlUrl
+  }}
+  <label>
+    {{t "components.layers-dialogs.settings.kml.string-textbox.caption"}}
+  </label>
+  {{flexberry-textbox
```

Comment:
If there are multiline mode in flexberry-textbox, it would be better

Reply:
fixed